docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py

```diff
@@ -289,7 +289,23 @@ def test_allowed_query_params(self):
             {
                 "interaction_id": "urn:nhs:names:services:cpisquery:REPC_IN000007GB01",
                 "party_key": "RTX-821088"
-            }
+            },
+            {
+                "org_code": " RTX",
+                "interaction_id": "urn:nhs:names:services:ebs:PRSC_IN070000UK08",
+            },
+            {
+                "org_code": "RTX ",
+                "interaction_id": "urn:nhs:names:services:ebs:PRSC_IN070000UK08",
+            },
+            {
+                "org_code": "RTX",
+                "interaction_id": " urn:nhs:names:services:ebs:PRSC_IN070000UK08",
+            },
+            {
+                "org_code": "RTX",
```

Comment:
These are duplicated? Shouldn't they all be unique?

Reply:
It's testing accepted params, this is all, there are spaces inserted into each so they are not exact duplicates

Reply:
Ah right! My skimming didn't see that 😅